extensions/wikia/ARecoveryEngine/ARecoveryEngineApiController.class.php

```diff
@@ -0,0 +1,16 @@
+<?php
+class ARecoveryEngineApiController extends WikiaController {
+
+	const DEFAULT_TEMPLATE_ENGINE = WikiaResponse::TEMPLATE_ENGINE_MUSTACHE;
+
+	public function getDelivery() {
+
+
+		$resourceLoader = new ResourceLoaderAdEngineSourcePointCSDelivery();
```

Comment:
I wonder how could we decouple it with `ResourceLoaderAdEngineSourcePointCSDelivery` and `ResourceLoaderContext`.